verified_model/Site/South/altitude.json

```diff
@@ -0,0 +1,75 @@
+{
+    "Value": 2157.0,
+    "units": "m",
+    "Type": "numpy.float64",
+    "File": false,
+    "Metadata": {
+        "CTA": {
+            "REFERENCE": {
+                "VERSION": "1.0.0"
+            },
+            "CONTACT": {
+                "ORGANIZATION": "CTAO",
+                "NAME": "CTAO Simulation Team",
+                "EMAIL": null
+            },
+            "PRODUCT": {
+                "DESCRIPTION": "Altitude above sea level of site centre.",
```

Comment:
I am a bit worried about the duplication here with respect to the schema. Is this the entry by the user?

Reply:
This is correct and it also caught my eye this morning. 

PRODUCT:DESCRIPTION is a required entry in the CTA data model, saying "Human-readable description of data product". Not sure what the best solution is, you are absolutely right with the duplication.